CooperationHullMainSite/Views/Shared/_footer.cshtml

```diff
@@ -1,10 +1,24 @@
-﻿        <div class="row">
-            <div class="col-lg-6 col-md-12">
-                <h3>Cooperation Hull 2024</h3>
+﻿<footer class="homepage-block p-3">
+    <div class="footer-content">
+        <div class="footer-content-left">
+            <p class="heading-md">Cooperation Hull ©️ 2024</p>
+            <div class="footer-links" role="navigation">
+                <a asp-area="" asp-controller="Home" asp-action="PrivacyPolicy">Privacy</a>
+                <a asp-area="" asp-controller="Home" asp-action="ContactUs">Contact Us</a>
+                <a asp-area="" asp-controller="Home" asp-action="CookiePolicy">Cookies Policy</a>
             </div>
-        <div class="col-lg-6 col-md-12 footer-links" role="navigation" aria-label="Footer">
-                <p> <a asp-area="" asp-controller="Home" asp-action="PrivacyPolicy">Privacy</a></p>
-                <p> <a asp-area="" asp-controller="Home" asp-action="ContactUs">Contact Us</a></p>
-                <p> <a asp-area="" asp-controller="Home" asp-action="CookiePolicy">Cookies Policy</a></p>
-            </div>
-        </div>
\ No newline at end of file
+        </div>
+        <div class="footer-socials">
+            <a asp-area="" asp-controller="Home" asp-action="Facebook">
```

Comment:
links done this way only work for internal pages set up in the relevant controllers.
For going to external pages then easiest way is to use the standard  <a href='www.facebook.com' target='_blank'> type.
(Unless you want to add in a controller action for each that then redirects to the relevant external site, or add in the external url's to setting, but as they are not likely to change I think that'd be overkill for these ones)